client/gutenberg/editor/controller.js

```diff
@@ -316,23 +316,21 @@ export function redirectToPermalinkIfLoggedOut( context, next ) {
 		return next();
 	}
 	const siteFragment = context.params.site || getSiteFragment( context.path );
-	const CONFIGURABLE_TYPES = [ 'jetpack-portfolio', 'jetpack-testimonial' ];
-
-	// The context.path in this case could be one of the following:
+	if ( ! siteFragment || ! context.path ) {
+		return next();
+	}
+	// Check the path is this format.
 	// - /page/{site}/{id}
 	// - /post/{site}/{id}
 	// - /edit/jetpack-portfolio/{site}/{id}
 	// - /edit/jetpack-testimonial/{site}/{id}
```

Comment:
I guess we can remove these since we're not specifically targeting any specific post type anymore, but I guess it doesn't hurt to be verbose too.

Reply:
I would leave it since it is a good reminder of the different custom post types that we support

Reply:
love keeping the examples in, but now the comment and code are awkward 🙃 

might be good to at least change the first line to something like this

```js
// "single view" pages are parsed from URLs like these:
// (posts, pages, custom post types, etc…)
//  - /page/{site}/{id}
//  - /post/{site}/{id}
//  - /edit/jetpack-portfolio/{site}/{id}
//  - /edit/jetpack-testimonial/{site}/{id}
```

could be a lovely quick follow-up